### ISSUE TITLE
Laravel 5.1.20 Compatibility

### ISSUE THREAD

**Issue by BaoHung** [Vinelab/NeoEloquent#119]
I'm not sure if this is Laravel's issue or NeoEloquent's since 5.1.19 still work well.

Here's the log when I access Neo4j DB:

exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function processSelect() on null' in vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php:1383
Stack trace:
#0 {main}

**Comment by heydavid713** [comment]
I was looking into it yesterday as it also happened to us. The issue seems to be that Illuminate\Database\Query\Builder changed the methods get and getFresh.

**Comment by fesja** [comment]
I've been trying Laravel+NeoEloquent on a new site (Laravel 5.1.20) and it also happens to me.

**Comment by tomb10516** [comment]
I looked in to this a bit, and got individual test cases working (see bottom), but don't have time to do a real fix, I was only experimenting to see where in general the problem is.  I thought I'd share what I saw so that someone else can fix faster.

The problem has to do with the fact that the Connection class now needs to have a $postProcessor set.  The NeoEloquent constructors and other ways of creating Connections don't always set this field.

So I got individual tests working by creating a default post processor, for example in the \NeoEloquent\Connection constructor I added 

                $this->useDefaultPostProcessor();

because that line of code is in the overridden constructor and overriding constructor (NE\Connection::__construct) does not call its parent.

Should it be calling its parent perhaps?  Anyway there are other places where a post processer must be set such as NeoEloquent\Query\Builder

The first test that fails when running 1202bd8 with illuminate/database v5.1.20  is testAddDropLabelsRepeatedlyOnModelsFoundById

If you checkout https://github.com/tomb10516/NeoEloquent/tree/l-5.1.20-experiment you will see that test passes.
